ap_src/ap_site/settings.py

```diff
@@ -105,17 +105,6 @@
     }
 }
 
-if not DEBUG:
```

Comment:
This code block was removed because of an issue with the `.env` that I was advised to remove to fix said issue. The problem was that the shell needed rebooting and that has since been fixed.

To address this, I will add the removed code back in.

Fixed with 027e723